src/Renderer/Shader/TileFS.glsl

```diff
@@ -12,13 +12,12 @@ const vec4 COrange = vec4( 1.0, 0.3, 0.0, 1.0);
 const vec4 CRed = vec4( 1.0, 0.0, 0.0, 1.0);
 
 
-uniform sampler2D   dTextures_01[TEX_UNITS];
-uniform vec4        offsetScale_L01[TEX_UNITS];
+uniform sampler2D   texColor_01[TEX_UNITS];
```

Comment:
Good idea to rename all the variables to better names.

This could be improved though:
- texColor_01 -> colorTextures
- offsetScale_color_01 -> colorOffsetScale or colorTexturesOffsetScale
- paramLayers -> layerParameters or layerParams
- texElevation[1] -> elevationTexture
- offsetScale_elevation -> elevationTextureOffsetScale

And this change alone could be moved to a cleanup PR.